admin/aioseop_module_class.php

```diff
@@ -855,8 +858,8 @@ function get_taxonomy_titles( $args = array() ) {
 		 *
 		 * @return array
 		 */
-		function get_category_titles( $args = array() ) {
-			return $this->get_term_labels( get_categories( $args ) );
+		function get_category_titles( $args = array(), $prefixed_with_taxonomy = false ) {
```

Comment:
PHPDoc needs to be updated, and noting a change in `since`.

Reply:
done